data/ddl.sql

```diff
@@ -1,93 +1,131 @@
-create table users (
-    id uuid primary key,
-    first_name text not null ,
-    last_name text,
-    email text not null,
-    password_digest text
+create table users
+(
+  id uuid primary key,
+  first_name text not null ,
+  last_name text,
+  email text not null,
+  password_digest text,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP
```

Comment:
We're using lowercase keywords in this file, can you adapt it to follow the pattern?
```suggestion
  updated_at timestamp default CURRENT_TIMESTAMP
```

Reply:
Addressed the comment. Kindly verify.